### ISSUE TITLE
Jei Overlap

### ISSUE THREAD

**Issue by WenXin20** [ToroCraft/Dailies#39]
When a quest is accepted, the quest appears under the jei menu. 
Maybe you could add a way to customize the quests' position or make it so jei detects it so that it's menu will go around the quest.

**Comment by pandapaul** [comment]
Absolutely.  This overlaps with #36, which very much needs to get addressed.

**Comment by WenXin20** [comment]
Thanks

**Comment by Iteries** [comment]
Like an updated progress as JEI with Dailies behind it makes it very difficult make out what object/items are there. Suggest moving the quest or make it invisible when JEI is up?  Either that or have open quests as an item on red-paper.  Once complete the paper turns green so peeps can turn it in.

**Comment by coldblacksnow** [comment]
Can you add button to switch visibility of quests in the inventory?

**Comment by pandapaul** [comment]
@sonofthesnow There should be an option under mod configuration to allow toggling the visibility of quests in inventory.  Should read `Show Quests In Inventory`.

**Comment by pandapaul** [comment]
@Iteries This is still a high priority fix.  We've just been working on a lot of other mods lately.  It'll happen eventually though, I'm sure. :)  Sorry for the incredibly slow response.  I missed the github email notification apparently.

**Comment by Wing01** [comment]
We are currently working on the ability to map the dailies gui to a button that can be set in the mod config. You won't have to rely on the inventory screen to see your accepted quests. Will be available for 1.12+. Stay tuned.

**Comment by CalaMariGold** [comment]
Any word on this @Wing01 @pandapaul? Would like to know if the mod is still in development.

**Comment by pandapaul** [comment]
I don't think we've made progress on this one, and generally Dailies hasn't
moved much in a while.  Still it's my personal favorite mod to work on.
Just haven't had the chance to do so.

**Comment by CalaMariGold** [comment]
@pandapaul Are we allowed to work on this mod ourselves and publish it as a separate mod, so long as it stays open source? I'm really looking to use Toroquest and this mod but both mods need a lot of tweaking.

**Comment by frodare** [comment]
@CalaMariGold  Yes, since it is GPL3, if you keep it open source you are good, same with Toroquest.
